### PR TITLE
Translate to Portuguese: examples/mp-config-source-database

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira:https://issues.apache.org/jira/browse/TOMEE-2829

### CODE REVIEW

File: examples/mp-config-source-database/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,327 @@
+= Banco de dados ConfigSource da configuração do MicroProfile
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+Este es un ejemplo sobre cómo implementar un ConfigSource para la configuración personalizada de MicroProfile. El ConfigSource personalizado lee los valores 
+de configuración de una base de datos.
+
```
Comment (hbelmiro) [inline examples/mp-config-source-database/README_pt.adoc:8]: Remove those lines

File: examples/mp-config-source-database/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+Este é um exemplo sobre como implementar um `ConfigSource` para a configuração personalizada de MicroProfile.
+O ConfigSource personalizado le os valores de configuração de uma base de dados.
```
Comment (hbelmiro) [inline examples/mp-config-source-database/README_pt.adoc:10]:
```suggestion
O ConfigSource personalizado lê os valores de configuração de uma base de dados.
```

File: examples/mp-config-source-database/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+
+[discrete]
+==== ConfigSource Feature
+
+Para proporcionar um ConfigSource personalizado de uma base de dados, devemos começar com uma classe que implemente la interface `ConfigSource`, sobreescrevendo 3 métodos: `getProperties`, `getValue`, e `getName`.
```
Comment (hbelmiro) [inline examples/mp-config-source-database/README_pt.adoc:15]:
```suggestion
Para proporcionar um ConfigSource personalizado de uma base de dados, devemos começar com uma classe que implemente a interface `ConfigSource`, sobrescrevendo 3 métodos: `getProperties`, `getValue`, e `getName`.
```

File: examples/mp-config-source-database/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+        return null;
+    }
+
+    @Override
+    public String getName() {
+        return DatabaseConfigSource.class.getSimpleName();
+    }
+}
+----
+Para simplificar, no exemplo anterior, a definição da base de dados e os dados que se utilizaram para a configuração correspondem a um `Resource`, declarado em um arquivo de configuração `tomee.xml` como `Resource` do tipo: `DataSource` da seguinte maneira:
```
Comment (hbelmiro) [inline examples/mp-config-source-database/README_pt.adoc:82]:
```suggestion
Para simplificar, no exemplo anterior, a definição da base de dados e os dados utilizados para a configuração correspondem a um `Resource`, declarado em um arquivo de configuração `tomee.xml` como `Resource` do tipo: `DataSource` da seguinte maneira: 
```

File: examples/mp-config-source-database/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+[source,xml]
+----
+<tomee>
+  <Resource id="config-source-database" type="DataSource">
+
+  </Resource>
+</tomee>
+----
+
+e esta vinculado ao conjunto de instruções SQL, definidas no script `import-config-source-database.sql`:
```
Comment (hbelmiro) [inline examples/mp-config-source-database/README_pt.adoc:93]:
```suggestion
e está vinculado a um conjunto de instruções SQL, definidas no script `import-config-source-database.sql`:
```

File: examples/mp-config-source-database/README_pt.adoc (hunk 1 above)
Comment (Daniel-Dos) [reply examples/mp-config-source-database/README_pt.adoc]: done thanks

File: examples/mp-config-source-database/README_pt.adoc (hunk 2 above)
Comment (Daniel-Dos) [reply examples/mp-config-source-database/README_pt.adoc]: done thanks

File: examples/mp-config-source-database/README_pt.adoc (hunk 4 above)
Comment (Daniel-Dos) [reply examples/mp-config-source-database/README_pt.adoc]: done thanks

File: examples/mp-config-source-database/README_pt.adoc (hunk 5 above)
Comment (Daniel-Dos) [reply examples/mp-config-source-database/README_pt.adoc]: done thanks

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro for review .

**Pull request merged**